homeassistant/components/youtube/coordinator.py

```diff
@@ -70,4 +70,8 @@ async def _async_update_data(self) -> dict[str, Any]:
                 }
         except UnauthorizedError as err:
             raise ConfigEntryAuthFailed from err
+        except YouTubeBackendError as err:
+            if self.last_update_success:
+                LOGGER.warning("Couldn't connect to YouTube")
```

Comment:
Just include the message as an exception argument to `UpdateFailed` instead. It will be logged by the coordinator and the coordinator will make sure not to spam the log.